extensions/wikia/CommunityPage/scripts/ext.communityPage.js

```diff
@@ -184,10 +184,5 @@ require([
 	$(function () {
 		// prefetch UI modal on DOM ready
 		getUiModalInstance();
```

Comment:
Yes, because the throbber is shown inside the modal (when loading tab data), so having the actual modal preloaded will avoid having to put up a throbber for loading the modal too.